apps/workshop/projects/DavosProtocol/index.ts

```diff
@@ -0,0 +1,18 @@
+import type { ProjectDefinitionInterface } from '@defiyield/sandbox';
+
+import { DavosProtocol } from './modules/DUSDStake';

```

Comment:
looks like a missing reference

Reply:
on the second line?

Reply:
but the example module has `abi` inside, whereases the module in the PR doesn't.  Please make sure the module works fine running:
 yarn workshop start [integration_name]

Reply:
apart from the abis module, is there anything else required?